### ISSUE TITLE
Resnet overfitting

### ISSUE THREAD

**Issue by shuuchen** [yunjey/pytorch-tutorial#127]
Hi there,

I tried to implement resnet and put it into Jupyter Notebook.
https://github.com/shuuchen/pytorch_tutorials/blob/master/Deep%20Residual%20Network/drn.nbconvert.ipynb

However in the final gragh, the model got overfitting terribly, where the training loss getting nearly 0 but the testing loss getting higher and higher.

Anyone can help?

**Comment by fawazsammani** [comment]
As I can see from your code, you haven't normalized your images in the transforms. It happened once with me, then when I normalized them, overfitting was reduced. You need to find the mean and std deviation of your images, and then provide them to the data transforms. 
Hope that helps

**Comment by shuuchen** [comment]
@fawazsammani 
Thanks very much !
I used BatchNorm2d layers for normalization. Don' t they do what you talked about ?

**Comment by tooHotSpot** [comment]
> @fawazsammani
> Thanks very much !
> I used BatchNorm2d layers for normalization. Don' t they do what you talked about ?

Check BatchNorm parameter track_running_stats, it has to be True, by default it is False

**Comment by shuuchen** [comment]
@tooHotSpot 

> by default it is False
Actually, it is True by default
